fluent-resmgr/tests/localization_test.rs

```diff
@@ -36,11 +36,13 @@ fn localization_format_value() {
 fn resmgr_get_bundle() {
     let res_mgr = ResourceManager::new("./tests/resources/{locale}/{res_id}".into());
 
-    let bundle = res_mgr.get_bundle(vec![langid!("en-US")], vec!["test.ftl".into()]);
+    let bundle = res_mgr
+        .get_bundle(vec![langid!("en-US")], vec!["test.ftl".into()])
+        .expect("Could not get bundle");
 
     let mut errors = vec![];
     let msg = bundle.get_message("hello-world").expect("Message exists");
     let pattern = msg.value().expect("Message has a value");
-    let value = bundle.format_pattern(&pattern, None, &mut errors);
```

Comment:
Question: These `&` changes seem unrelated drive-by fixes. I'm assuming since the `pattern` here is already a reference.

Reply:
Yeah, I did a few `clippy` quick fixes, I thought it wouldn't hurt much 😄